### PR TITLE
add infrastructure to mirror all bioconda tarballs

### PR DESCRIPTION
**Pull request by bgruening**
This will add the initial infrastructure to use Jenkins to mirror all conda (currently bioconda) tarballs on depot.galaxyproject.org.

Currently `linux64` and `osx` is supported and will be archived.

### CODE REVIEW

File: .conda/get_meta.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+
+DIRECTORY=./bioconda-recipes
+
+if [ -d "$DIRECTORY" ]; then
+    cd bioconda-recipes
```
Comment (erasche) [inline .conda/get_meta.sh]: `cd $DIRECTORY`

File: .conda/install.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+CONDA_INSTALLATION_PATH=$HOME/anaconda
```
Comment (erasche) [inline .conda/install.sh]: should we consider changing this to `$CWD` or so? I don't want to pollute/be a bad citizen of the jenkins server and place things outside our workspace

File: .conda/install.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+if hash conda 2>/dev/null; then
+    echo "Conda already installed"
+else
+    date "Installing Conda"
+    curl -O https://repo.continuum.io/miniconda/Miniconda2-latest-Linux-x86_64.sh
+    bash Miniconda2-latest-Linux-x86_64.sh -b -p CONDA_INSTALLATION_PATH/anaconda
+    export PATH=CONDA_INSTALLATION_PATH/anaconda/bin:$PATH
```
Comment (erasche) [inline .conda/install.sh]: should these be `$CONDA_INSTALLATION_PATH`?

File: .conda/install.sh (hunk 2 above)
Comment (bgruening) [inline .conda/install.sh]: I would use /tmp/ on Jenkins I think.

File: .conda/install.sh (hunk 3 above)
Comment (bgruening) [inline .conda/install.sh]: Indeed.

**Comment by bgruening** [comment]
Yes!!!

**Pull request merged**